app/assets/stylesheets/chart.scss

```diff
@@ -1,7 +1,13 @@
-.chart {
-	rect {
-    fill: white;
-    stroke: black;
-    stroke-width: 1px;
-	}
+.active-bookings {
+  fill: white;
```

Comment:
At some point we should define a colorscheme to use on charts throughout the site. We can take care of this then.